cdi-unit/src/main/java/org/jglue/cdiunit/internal/TestConfiguration.java

```diff
@@ -0,0 +1,55 @@
+package org.jglue.cdiunit.internal;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Defines, how the WeldTestUrlDeployment should initialize Weld.
+ *
+ * @author aschoerk
+ */
+public class TestConfiguration {
+
+    public TestConfiguration(Class<?> testClass, Method testMethod, Collection<Class<?>> additionalClasses) {
+        this.testClass = testClass;
+        this.testMethod = testMethod;
+        this.additionalClasses = additionalClasses != null ? additionalClasses : new ArrayList<>();
```

Comment:
Okay, robustness versus early discovery of an error. The latter is preferable. I am a bit reluctant, to introduce findbugs just for that, so I put a check inside the constructor